docs/mocserver/mocserver.rst

```diff
@@ -11,433 +9,383 @@ Getting started
-then the MOCServer knows that some sources of this data-set are in the user defined sky region.
+The idea is the same for Time MOCs. The time axis is split into time cells.
 
-To be aware of what the MOCServer returns, please refers to this `link
-<http://alasky.unistra.fr/MocServer/query?RA=10.8&DEC=32.2&SR=1.5&intersect=overlaps&get=record&fmt=html>`_.
-We have queried the MOCServer with a cone region of center ra, dec = (10.8, 32.2) deg and radius = 1.5 deg. In return,
-the MOCServer gives a list of data-sets each tagged with an unique ID along with some other meta-datas too e.g.
-``obs_title``, ``obs_description``, ``moc_access_url`` (url for accessing the MOC associated with the data-set. Usually
-a FITS file storing a list of HEALPix cells).
+For those wanting to know more about MOCs, please refer to `the MOC 2.0 specification 
+document <https://ivoa.net/documents/MOC/20220727/REC-moc-2.0-20220727.pdf>`_.
 
-It is also possible to ask the MOCServer for retrieving data-sets based on their meta-data values. `Here
-<http://alasky.unistra.fr/MocServer/query?RA=10.8&DEC=32.2&SR=1.5&intersect=overlaps&get=record&fmt=html&expr=(dataproduct_type=image)>`_
-we have queried the MOCServer for only the image data-sets being in the cone defined above (``dataproduct_type``
-meta-data equals to ``"image"``).
+MOCPy is a Python library allowing to manipulate these Space-Time Coverage objects.
+You're welcome to have a look at `MOCPy's documentation <https://github.com/cds-astro/mocpy>`_.
 
-This package implements two methods:
+What's the MOC Server?
+----------------------
 
-* :meth:`~astroquery.mocserver.MOCServerClass.query_region` retrieving data-sets (their associated MOCs and meta-datas) having sources in a given region.
-* :meth:`~astroquery.mocserver.MOCServerClass.find_datasets` retrieving data-sets (their associated MOCs and meta-datas) based on the
-  values of their meta-datas.
+The MOC Server is a service of astronomical resources organized by spatial and/or
+temporal coverages following the Space and Time MOC specification.
+In the MOC Server, there a few tens of thousands of astronomical collections.
+They each have an identifier ``ID`` and a set of properties that describe their content.
+This is a practical way of finding datasets with criteria on time and/or space.
 
-Requirements
-----------------------------------------------------
-The following packages are required for the use of this module:
+The meta-data properties are freely assigned by each publisher. You can get the list of
+properties with their frequency of usage and examples example with 
+`astroquery.mocserver.MOCServerClass.list_fields`.
+This method also accepts a string to limit the number of responses. Let's try with ``MOC``:
+
+..
+   We ignore output here, as occurrence is changing often
+.. doctest-remote-data::
+
+  >>> from astroquery.mocserver import MOCServer
```

Comment:
These require mocpy (and also maybe regions) to be installed. You can use `.. doctest-requires:: mocpy` to skip the tests when they are not available. Unfortunately it has to be repeated for each and every code example as we don't have a global directive yet.

Reply:
Ah thanks!